nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -141,6 +141,11 @@
             <groupId>com.hierynomus</groupId>
             <artifactId>sshj</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.exceptionfactory.socketbroker</groupId>
```

Comment:
Is a NOTICE needed for this library?

Reply:
Thanks for checking. The library uses the standard Apache Software License 2.0 and does not include a separate NOTICE.  According to the Apache Infrastructure guidelines, additional NOTICE details should not be necessary:

https://infra.apache.org/licensing-howto.html

> The NOTICE file is reserved for a certain subset of legally required notifications which are not satisfied by either the text of LICENSE or the presence of licensing information embedded within the bundled dependency. Aside from Apache-licensed dependencies which supply NOTICE files of their own, it is uncommon for a dependency to require additions to NOTICE.